interface/reports/lab_stats_by_demographics_report.php

```diff
@@ -236,13 +236,13 @@ function lab_result_details() {
     <thead>
 
     <tr>
-        <th><input  id = 'column0_search_show_lab_details_table' size="4"></th>
```

Comment:
https://github.com/LibreHealthIO/lh-ehr/pull/1284/files should not affect this, right?  It is another PR coming up.  Probably needs to be merged first, as that is for the race condition.